src/odemis/dataio/tiff.py

```diff
@@ -2613,3 +2624,121 @@ def _iterDirectories(tiff_file):
             tiff_file.ReadDirectory()
             dir_index += 1
             yield dir_index
+
+
+ 
+### FEI Metadata parsing
+def _parseINIMetadataAsDict(metadata: str, delimeter: str = "\r\n"):
```

Comment:
Isn't it possible to just use the `configparser` module from the standard library?

Reply:
I attempted this initially, but couldn't get the parser to return anything except None. Tried different encodings and removing the returnline chars, but not sure. Open to suggestions

```python
import configparser
cfg = configparser.ConfigParser()
metadata = metadata.decode("latin1") # latin1 -> every char is acceptable
tfs_metadata = cfg.read_string(metadata)
```
tfs_metadata is None